cdap-app-fabric/src/main/java/io/cdap/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -345,6 +349,12 @@ protected void configure() {
       // Used in InMemoryProgramRunDispatcher, TetheringClientHandler
       install(RemoteAuthenticatorModules.getDefaultModule(TetheringAgentService.REMOTE_TETHERING_AUTHENTICATOR,
                                                           Constants.Tethering.CLIENT_AUTHENTICATOR_NAME));
+      // TODO: CDAP-20322 add the remoteSourceControlOperationRunner
+      bind(SourceControlOperationRunner.class).to(InMemorySourceControlOperationRunner.class).in(Scopes.SINGLETON);
```

Comment:
should have a guice module in the sourcecontrol part of the code instead of having bindings here, as the sourcecontrol section is where all the relevant classes live. This module would use that SourceControlModule in getInMemoryModules(), etc.

Reply:
Created `SourceControlModule`